pkg/occlient/occlient.go

```diff
@@ -1,15 +1,17 @@
 package occlient
 
 import (
+	taro "archive/tar"
```

Comment:
because there is conflict of name `tar`, we are using tar from `S2I` as well,
https://github.com/surajnarwade/ocdev/blob/28c3a6c56db39ad593a0cc79b099eea9d1d24535/pkg/occlient/occlient.go#L24